packages/flame/lib/src/components/position_component.dart

```diff
@@ -402,4 +403,14 @@ class PositionComponent extends Component {
     size.setValues(rect.width, rect.height);
     topLeftPosition = rect.topLeft.toVector2();
   }
+
+  //#region Effect interfaces
```

Comment:
There will be a lot of effect specific code in here if we continue down this path.
if `PositionProvider` "looks" for the `position` field instead at least these overrides shouldn't be necessary.

Reply:
Alright, I'll make the interface refer to the `position` field instead.